README.md

```diff
@@ -1,2 +1,84 @@
-# q2_g88
-Community Web-App project for Galvanize WDI Q2.
+# Fam Web App
+A community Web-App built by Galvanize WDI students as a Q2 project.
+
+Everyone needs a community. Build yours using the Fam Web App. Connect with other parents in your area, lend a helping hand to a new mom in need of a little house cleaning or a meal. Ask for help from local parents to come walk your dog or just chat with you because it can be pretty isolating being a stay at home parent.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+Give examples and Photos
+```
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+Say what the step will be
+
+```
+Give the example in the form of screenshots
+```
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+<!-- ## Running the tests
+
+Explain how to run the automated tests for this system
+
+### Break down into end to end tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+### And coding style tests
+
+Explain what these tests test and why
+
+```
+Give an example
+``` -->
+
+## Deployment
+
+Add additional notes about how to deploy this on a live system
+
+## Built With
+
+* [Bootstrap](https://startbootstrap.com/template-overviews/freelancer/) - The frontend framework used.
+* [?](weblink) - Dependency Management
+* [Knex](weblink) - Used to generate database and backend structure.
+<!-- do we need one for web framework used? IS that the same as bootstrap? -->
+
+## Authors
+
+* **Brechin Andariese** [Brechy](https://github.com/Brechy)
+
+[NikSS](https://github.com/niksseif)
+
+[JBallin](https://github.com/JBallin)
+
+[Contributors](https://github.com/Brechy/q2_g88/graphs/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
```

Comment:
@Brechy does his template have a license file we can pull in? Currently we don't have one.

Reply:
I was going to add it otherwise we can remove that line. I removed a few other things that were not relevant to our project. I will add the generic MIT license that github has available. That is what I did when I was playing around with code of conduct templates 😄

Reply:
im indifferent on adding file vs. removing from readme. but we need to do one.

Reply:
adding the generic github one works!